projects/transformers/finetuning_constants.py

```diff
@@ -46,7 +47,8 @@
     "qnli": "QNLI",
     "rte": "RTE",
     "wnli": "WNLI",
-    "": "AX"  # diagnostic set, not yet implemented
+    "": "AX",  # diagnostic set, not yet implemented
```

Comment:
what is AX?

Reply:
A diagnostic dataset we haven't used so far that is included in the GLUE datasets, but not factored into the the GLUE score. Last row of this table: https://gluebenchmark.com/tasks.